### PR TITLE
Added Social Links

### PR DESCRIPTION
**Pull request by NaviRocker**
Added Social Links to Speakers
#71

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -663,7 +665,7 @@ <h1>Featured Speakers</h1>
 									<div class="hover-state text-center preserve3d">
 										<div class="social-links vertical-align">
 											<a target="default" href="https://developers.google.com/open-source/"><i class="icon fa fa-external-link"></i></a>
-
+                                            <a target="default" href="https://twitter.com/StephanieTaylor?lang=en"><i class="icon social_twitter"></i></a>
```
Comment (vaibhavsingh97) [inline index.html]: Clean the link

File: index.html [hunk 2]
```diff
@@ -733,7 +735,7 @@ <h1>Featured Speakers</h1>
 									<div class="hover-state text-center preserve3d">
 										<div class="social-links vertical-align">
 											<a target="default" href="https://wiki.freebsd.org/PhilipPaeps"><i class="icon fa fa-external-link"></i></a>
-
+                                            <a target="default" href="https://www.linkedin.com/company/940332/"><i class="icon social_linkedin"></i></a>
```
Comment (vaibhavsingh97) [inline index.html]: Aren't they have linked in profile?

File: index.html [hunk 3]
```diff
@@ -1293,7 +1295,7 @@ <h1>Featured Speakers</h1>
 									<div class="hover-state text-center preserve3d">
 										<div class="social-links vertical-align">
 											<a target="default" href="https://pep.foundation/"><i class="icon fa fa-external-link"></i></a>
-
+                                            <a href="https://www.linkedin.com/company/4525407/" target="default"><i class="icon social_linkedin"></i></a>
```
Comment (vaibhavsingh97) [inline index.html]: Same question

**Review by vaibhavsingh97 — CHANGES_REQUESTED** [review]

**Comment by simsausaurabh** [comment]
Please ensure that github icon shows up, or else if doesn't show
Use `<i class="fa fa-github" aria-hidden="true"></i>`
and add `<script src="https://use.fontawesome.com/40afd82a34.js"></script>` at `<head>` section

File: index.html (hunk 3 above)
Comment (NaviRocker) [reply index.html]: @vaibhavsingh97 I searched for their Official Linked In Profile. But I was not able to find them. As they have added their Company/Organization Link for other Social Media, I added Their Company/Organization link.

**Comment by NaviRocker** [comment]
@simsausaurabh I didn't find any problem with Icons. They are fine.

**Review by 0x48piraj — CHANGES_REQUESTED** [review]
Do the appropriate changes.
Pay attention to existing reviews before asking for new one.
Also, keep an eye on the links as @vaibhavsingh97 suggested :+1:

**Comment by NaviRocker** [comment]
@vaibhavsingh97 @0x48piraj Fixed

File: index.html [hunk 4]
```diff
@@ -649,8 +651,8 @@ <h1>Featured Speakers</h1>
 									<div class="hover-state text-center preserve3d">
 										<div class="social-links vertical-align">
 											<a target="default" href="http://metafluff.com/"><i class="icon fa fa-external-link"></i></a>
-                                            						<a target="default" href="https://github.com/autonome"><i class="icon fa fa-github"></i></a>
```
Comment (nzec) [inline index.html:652]: Fix indentation.

**Comment by NaviRocker** [comment]
Fixed @duttaditya18

File: index.html [hunk 5]
```diff
@@ -600,6 +600,7 @@ <h1>Featured Speakers</h1>
 									<div class="hover-state text-center preserve3d">
 										<div class="social-links vertical-align">
 											<a target="default" href="https://twitter.com/0rb1t3r"><i class="icon social_twitter"></i></a>
+                                            <a target="default" href="https://github.com/orbiter/"><i class="icon fa fa-github"></i></a>
```
Comment (akshatnitd) [inline index.html:603]: Indent issues

File: index.html [hunk 6]
```diff
@@ -633,6 +634,7 @@ <h1>Featured Speakers</h1>
 									<div class="hover-state text-center preserve3d">
 										<div class="social-links vertical-align">
 											<a target="default" href="http://www.credativ.com/"><i class="icon fa fa-external-link"></i></a>
+                                            <a target="default" href="https://github.com/bunnie"><i class="icon fa fa-github"></i></a>
```
Comment (akshatnitd) [inline index.html:637]: Indent issues

File: index.html [hunk 7]
```diff
@@ -667,6 +669,7 @@ <h1>Featured Speakers</h1>
 									<div class="hover-state text-center preserve3d">
 										<div class="social-links vertical-align">
 											<a target="default" href="https://developers.google.com/open-source/"><i class="icon fa fa-external-link"></i></a>
+                      <a target="default" href="https://twitter.com/StephanieTaylor"><i class="icon social_twitter"></i></a>
```
Comment (akshatnitd) [inline index.html:672]: Indent issues

File: index.html [hunk 8]
```diff
@@ -1295,8 +1297,7 @@ <h1>Featured Speakers</h1>
 									<div class="hover-state text-center preserve3d">
 										<div class="social-links vertical-align">
 											<a target="default" href="https://pep.foundation/"><i class="icon fa fa-external-link"></i></a>
-
-										</div>
+                                        </div>
```
Comment (akshatnitd) [inline index.html:1300]: Indent issues

File: index.html [hunk 9]
```diff
@@ -1329,7 +1330,7 @@ <h1>Featured Speakers</h1>
 									<div class="hover-state text-center preserve3d">
 										<div class="social-links vertical-align">
 											<a target="default" href="https://www.wikimedia.org/"><i class="icon fa fa-external-link"></i></a>
-
+                                            <a target="default" href="https://www.linkedin.com/company/27378/"><i class="icon social_linkedin"></i></a>
```
Comment (akshatnitd) [inline index.html:1333]: Indent issues

File: index.html [hunk 10]
```diff
@@ -1451,7 +1452,7 @@ <h1>Featured Speakers</h1>
 									<div class="hover-state text-center preserve3d">
 										<div class="social-links vertical-align">
 											<a target="default" href="https://wiki.debian.org/AndrewLee"><i class="icon fa fa-external-link"></i></a>
-
+                                            <a target="default" href="https://www.linkedin.com/company/163400/"><i class="icon social_linkedin"></i></a>
```
Comment (akshatnitd) [inline index.html:1455]: Indent issues

**Review by akshatnitd — CHANGES_REQUESTED** [review]
Correct the overall indentation :)

**Pull request closed without merging**